tests/mvp-demo/mvp-demo.go

```diff
@@ -435,7 +436,7 @@ var _ = framework.MvpDemoSuiteDescribe("MVP Demo tests", Label("mvp-demo"), func
 					return false
 				}
 				return pipelineRun.HasStarted()
-			}, pipelineRunStartedTimeout, defaultPollingInterval).Should(BeTrue())
+			}, pipelineRunStartedTimeout, defaultPollingInterval).Should(BeTrue(), fmt.Sprintf("failed to get pipelinerun named %q in namesapce %q with label %q to start", release.Name, managedNamespace, release.Namespace))
```

Comment:
Shouldn't be `pipelineRun.Name`?

```suggestion
}, pipelineRunStartedTimeout, defaultPollingInterval).Should(BeTrue(), fmt.Sprintf("failed to get pipelinerun named %q in namespace %q with label %q to start", pipelineRun.Name, managedNamespace, release.Namespace))
```